types/checkout/checkout.d.ts

```diff
@@ -97,3 +97,5 @@ type CheckoutOpenOptionsWithDiscount = CheckoutOpenOptionsWithDiscountId | Check
 export type CheckoutOpenOptions = CheckoutOpenBaseOptions &
   CheckoutOpenOptionsWithLineItems &
   CheckoutOpenOptionsWithDiscount;
+
+export type CheckoutUpdateOptions = Omit<CheckoutOpenOptions, 'settings' | 'customData' | 'transactionId'>;
```

Comment:
Could consider doing it like this to be consistent with the other types in this file?

```typescript
export type CheckoutUpdateOptions = CheckoutOpenOptionsWithDiscount & {
  items: CheckoutOpenLineItem[];
  customer?: CheckoutCustomer;
}
```

(also looks good as it is)